src/Symfony/Component/AssetMapper/Tests/ImportMap/RemotePackageStorageTest.php

```diff
@@ -41,6 +41,21 @@ public function testGetStorageDir()
         $this->assertSame(realpath(self::$writableRoot.'/assets/vendor'), realpath($storage->getStorageDir()));
     }
 
+    public function testSaveThrowsWhenVendorDirectoryIsNotWritable()
+    {
+        if (!file_exists($vendorDir = self::$writableRoot.'/assets/acme/vendor')) {
```

Comment:
IIRC there is a new method on filesystem for that

Reply:
On 6.4 too ?

Reply:
Updated the tests